api/src/pcapi/routes/backoffice/collective_offers/collective_offers_blueprint.py

```diff
@@ -287,8 +307,8 @@ def _get_collective_offer_ids_query(form: forms.GetCollectiveOfferAdvancedSearch
 
 
 def _get_collective_offers(
-    form: forms.GetCollectiveOfferAdvancedSearchForm,
-) -> list[educational_models.CollectiveOffer]:
+    collective_offers_ids: list[int] | sa.orm.Query,
```

Comment:
On importe déjà `sa_orm` donc tu peux remplacer les `sa.orm`.
```suggestion
    collective_offers_ids: list[int] | sa_orm.Query,
```